src/stores/widgets/PVGeneratorWidgetStore.ts

```diff
@@ -0,0 +1,60 @@
+import {action, observable, makeObservable, computed} from "mobx";
+import {CARTA} from "carta-protobuf";
+import {AppStore} from "stores";
+import {IOptionProps} from "@blueprintjs/core";
```

Comment:
move this include up above carta-protobuf because it's third-party

Reply:
done.